.ci/Jenkinsfile

```
@@ -298,7 +306,17 @@ def generateFunctionalTestStep(Map args = [:]){
   if (isPR() || isUpstreamTrigger(filter: 'PR-')) {
     tags += pullRequestFilter
   }
-  def workerLabels = "${platform} && immutable && docker"
+
+  def goArch = "amd64"
+  def workerLabels = "${platform} && immutable"
+  if (platform == "arm64") {
+    goArch = "arm64"
+    // ARM is not ready as immutable workers
+    workerLabels = "arm"
```

The CI worker labels for the ARM64:
- 'aarch64'
- 'aws'
- 'ubuntu-1804-aarch64'
- 'arm' 

```suggestion
    workerLabels = "aarch64"
```